nupic/encoders/pass_through_encoder.py

```diff
@@ -49,49 +48,48 @@ def __init__(self, n, w=None, name="pass_through", forced=False, verbosity=0):
     self.encoders = None
     self.forced = forced
 
-  ############################################################################
+
   def getDecoderOutputFieldTypes(self):
     """ [Encoder class virtual method override]
     """
     return (FieldMetaType.string,)
 
-  ############################################################################
+
   def getWidth(self):
     return self.n
 
-  ############################################################################
+
   def getDescription(self):
     return self.description
 
-  ############################################################################
+
   def getScalars(self, input):
     """ See method description in base.py """
     return numpy.array([0])
 
-  ############################################################################
+
   def getBucketIndices(self, input):
     """ See method description in base.py """
     return [0]
 
-  ############################################################################
-  def encodeIntoArray(self, input, output):
+
+  def encodeIntoArray(self, inputVal, outputVal):
```

Comment:
@oxtopus Why the `Val` suffix? It seems `input` and `output` names are common everwhere in code.